src/components/TextLink.js

```diff
@@ -76,4 +76,10 @@ function TextLink(props) {
 TextLink.defaultProps = defaultProps;
 TextLink.propTypes = propTypes;
 TextLink.displayName = 'TextLink';
-export default TextLink;
+export default React.forwardRef((props, ref) => (
+    <TextLink
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        forwardedRef={ref}
```

Comment:
`forwardedRef` property is not used

Reply:
I think it's fine, as we don't need it but good to have for a `forwardedRef`.

Reply:
It means that the `ref` will be not working at all

Reply:
There is no need for it. `TextLink` originally does not handle `ref`. Using `ForwardRef` is just for prevent console error.

Reply:
@cubuspl42 Added `forwardedRef` prop.